File: text/count_vowels.rb
vowels = [ 'a', 'e', 'i', 'o', 'u' ]
print "Please enter a string to count vowels with: "
input = gets.chomp
puts "You entered: #{input}"
results = input.split("")

results.each do |c| 
    if vowels.include? c
        puts "Character found: #{c}"
    end
end

File: text/count-vowels.py
vowels = { 'a': 0, 'e': 0, 'i': 0, 'o': 0, 'u': 0 }
word = raw_input('Enter a word to count vowels for: ')

for c in word.lower():
    if c in vowels:
        vowels[c] += 1

print 'Total vowel counts'
for v in vowels:
    print v + ': ', vowels[v]
